### PR TITLE
Style/writing page

### PR DESCRIPTION
**Pull request by gjtrecny**
# Description

Please check the type of change your PR introduces:
- [ ] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

 
Add styling to Error Page and Writing Page
 

### Type of change

Please check the relevant option.

- [X] Non-Breaking Change
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Big Breaking change (this breaks everything and I am sorry)

# Where should reviewers start?

# How Has This Been Tested?

Please check the option(s) that best describe your tests. 

- [ ] New spec files were added
- [ ] Tests were added to existing spec files
- [ ] Some tests were changed
- [ ] No tests were changed
- [ ] All tests were changed (please explain what in God's name happened)

### Tests Description

- Please add further details of your tests here.

# Checklist:

- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have removed all unused code or commented why I left it
- [ ] I have deployed my changes to a test-heroku and they work
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

**Review by philalewis — APPROVED** [review]
Good general cleanup. I'm guessing we will need to set up an environment in our CI in order for out tests to pass since it was previously relying on heroku to have the environment already set up.

**Review by Jayquintana — APPROVED** [review]
good work!

File: src/Components/Timer/Timer.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const Timer: React.FC<Props> = ({ totalSeconds, saveWriting, writingInProgress,
         className={`timer-button ${colorClass}`}
         onClick={toggleTimer}>{formatTime(time)}
       </button>
-      <p>target: {formatTime(totalSeconds)}</p>
+      <p>Target Time: {formatTime(totalSeconds)}</p>
```
Comment (Alynn022) [inline src/Components/Timer/Timer.tsx:73]: Nice catch and cleans up the way this is displayed!

**Review by Alynn022 — APPROVED** [review]
Looks great!!

**Pull request merged**